Amplify/Core/Configuration/Internal/Amplify+Reset.swift

```diff
@@ -22,7 +22,12 @@ extension Amplify {
 
         let group = DispatchGroup()
 
-        for categoryType in CategoryType.allCases {
+        let manuallyResetCategories = [CategoryType.logging, .hub]
+        let remainingCategories = CategoryType.allCases.filter {
+            !manuallyResetCategories.contains($0)
+        }
+
+        for categoryType in remainingCategories {
```

Comment:
can we maybe use the `partition(by:)` method to get a partitioned array with `.logging` and `.hub` cases at the end of it?